styles/main.css

```diff
@@ -1,283 +1,314 @@
 }
-.col-1{
+.col-1 {
   float: left;
   width: 8.33333%;
 }
-.col-2{
+.col-2 {
   float: left;
   width: 16.66666%;
 }
-.col-3{
+.col-3 {
   float: left;
   width: 25%;
 }
-.col-4{
+.col-4 {
   float: left;
   width: 33.33333%;
 }
-.col-5{
+.col-5 {
   float: left;
   width: 41.66666%;
 }
-.col-6{
+.col-6 {
   float: left;
   width: 50%;
 }
-.col-7{
+.col-7 {
   float: left;
   width: 58.33333%;
 }
-.col-8{
+.col-8 {
   float: left;
   width: 66.66666%;
 }
-.col-9{
+.col-9 {
   float: left;
   width: 75%;
 }
-.col-10{
+.col-10 {
   float: left;
   width: 83.33333%;
 }
-.col-11{
+.col-11 {
   float: left;
   width: 91.66666%;
 }
-.col-12{
+.col-12 {
   float: left;
   width: 100%;
 }
-.offset-1{
+.offset-1 {
   margin-left: 8.33333%;
 }
-.offset-2{
+.offset-2 {
   margin-left: 16.66666%;
 }
-.offset-3{
+.offset-3 {
   margin-left: 25%;
 }
-.offset-4{
+.offset-4 {
   margin-left: 33.33333%;
 }
-.offset-5{
+.offset-5 {
   margin-left: 41.66666%;
 }
-.offset-6{
+.offset-6 {
   margin-left: 50%;
 }
-.offset-7{
+.offset-7 {
   margin-left: 58.33333%;
 }
-.offset-8{
+.offset-8 {
   margin-left: 66.66666%;
 }
-.offset-9{
+.offset-9 {
   margin-left: 75%;
 }
-.offset-10{
+.offset-10 {
   margin-left: 83.33333%;
 }
-.offset-11{
+.offset-11 {
   margin-left: 91.66666%;
 }
-.offset-12{
+.offset-12 {
   margin-left: 100%;
 }
+.hero-section {
+  min-height: 100vh;
+  padding-bottom: 170px;
+  background-image: url(/images/Background.jpg);
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+}
 .main-logo,
 .button-login,
 .main-nav,
-li{
+li {
   list-style-type: none;
   font-size: 14px;
   font-style: normal;
 }
-.button-login{
+.button-login {
+  margin-left: 50px;
   text-transform: uppercase;
   padding: 11px 28px;
   color: #fff;
   background-color: transparent;
   border: 1px solid #fff;
   border-radius: 6px;
 }
-.main-header{
-  padding-top: 55px;
+.main-header {
+  width: 100%;
+  position: absolute;
+  padding-top: 50px;
   font-style: normal;
   font-weight: 400;
 }
-.main-nav-list{
+.main-nav-list {
   margin-left: 0;
   margin-right: 0;
   padding-left: 0;
   padding-right: 0;
   list-style-type: none;
 }
-.main-nav-list-item{
+.main-nav-list-item {
   list-style-type: none;
   display: inline-block;
-  margin-left: 15px;
+  padding-left: 15px;
 }
-.numbers{
+.numbers {
+  display: block;
   margin-top: 30px;
   margin-bottom: 10px;
   font-weight: bold;
   font-size: 100px;
   color: rgba(16, 16, 16, 0.08);
 }
-.features-link-text{
-  color: #ED4C5C;
+.features-link-text {
+  color: #ed4c5c;
 }
-.image-technology{
+.image-technology {
+  text-align: center;
+  min-height: 70px;
   margin-top: -80px;
 }
-.image-features{
+.image-features {
   width: 100%;
 }
-.text-features{
+
+.text-features {
+  color: #fff;
   padding-top: 16px;
   padding-bottom: 16px;
 }
-.main-map{
+.main-map {
   height: 310px;
   width: 100%;
   border-radius: 8px;
 }
-.main-nav a{
+.main-nav a {
```

Comment:
Please, do not use this kind of selectors. No nesting, no tags. Only classes, please.